acceptance/src/routes.js

```diff
@@ -20,7 +22,9 @@ import TypographyPage from "./pages/TypographyPage";
 
 export default [
   { path: "/avatar/", component: AvatarPage, label: "Avatar" },
+  { path: "/mui/avatar/", component: MuiAvatarPage, label: "Mui Avatar" },
```

Comment:
Much like the comment above, I think all the `material-ui` routes info should stay together.